### PR TITLE
Added G6.MD file

### PR DESCRIPTION
**Pull request by TheDiabeticWay**

### CODE REVIEW

**Comment by TheDiabeticWay** [comment]
> _No description provided._

I have used S9 and Note 8 for over 6 months with 90% coverage I also got other information by asking the question on Xdrip  Facebook Group at https://www.facebook.com/groups/xDripG5/permalink/2766708616735949/     

this is my first ever pull request! so any tip will greatly help me thanks!

File: Documentation/Mobiles For G6.md [hunk 1]
```diff
@@ -0,0 +1,141 @@
...
+   
+## Tips
+<br>
+On Doogee / MTK phones disable Intelligent power saving standby in Android Settings -> Battery or you will only receive a reading every 10 minutes.<br>
+* * *
+
+## Any question's you might have or want to share your mobile settings with us that work with Xdrip please free to get in touch!<br>
+<br>
+
+[Contact US](https://www.thediabeticway.co.uk/index.php/en/about-us/contact-us.html) or Visit us on [Facebook](https://www.facebook.com/groups/xDripG5/)
```
Comment (kskandispersonal) [inline Documentation/Mobiles For G6.md]: What is this for?  I don't think this website is related to xDrip+.

**Comment by TheDiabeticWay** [comment]
Added Thediabetic way to Status Page

File: Documentation/Mobiles For G6.md (hunk 1 above)
Comment (TheDiabeticWay) [reply Documentation/Mobiles For G6.md]: Its a logo I created if it is needed to be used! I did it for a few videos I made. not important but if you care to want to use it you can! The web site is related to xdrip has it mine!

**Comment by Navid200** [comment]
@TheDiabeticWay We believe that any Android phone can be used reasonably well with G5 or G6.
The G5.MD file was archived for that reason.  I don't believe there is a reason to create a G6.MD.

Would you please close this PR?

Thanks

**Comment by Navid200** [comment]
Can we close this pull request please?

**Pull request closed without merging**